django-backend/fecfiler/reports/form_3x/models.py

```diff
@@ -16,6 +16,7 @@ class Form3X(models.Model):
         unique=True,
     )
     change_of_address = models.BooleanField(default=False, null=True, blank=True)
+    committee_name = models.TextField(null=True, blank=True)
```

Comment:
I think we'll need committee_name in the report model so that the other forms have it

Reply:
Added committee_name for form 24. We already had it form form 1M and 99, though it does feel like we should move it to a parent class for all forms, but there currently isn't one.

Reply:
@sasha-dresden It looks like all the current forms have committee_name, street_1, street_2, city, state, and zip. Let's move all of them out of the reports_form* tables and into the reports_report table.